src/test/java/com/logigear/LoginTest.java

```diff
@@ -10,11 +19,17 @@
 import page_objects.HomePage;
 import page_objects.LoginPage;
 
+import java.io.FileReader;
```

Comment:
what do you need this for? doesn't look like you are reading anything?

Reply:
Sorry , I will delete all import unnecessary in the last update